backend/entities/ef/PimsAccessRequestOrganization.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System;
-using System.Collections.Generic;
```

Comment:
This sounds good to me but unless I'm missing something would be pointless - this file being auto-generated. 
Aren't all files under `entities/ef` regenerated by EF whenever the DB schema changes @devinleighsmith?

Reply:
Yes, do not modify files in the ef* folder. If the generated files are generating warnings, you'll need to add a rule to ignore the ef folder.

Reply:
why not?

Reply:
This import in question is needed (I think) for **HashSet** which is used for collections of related entity (one-to-many, etc). Not sure that EF "scaffold" tool is so smart to only add the import when there is a collection in the generated entity. Most likely they hardcode the imports to cover all scenarios. 

In any case, we don't control what the tool generates so agree with Devin we need to add an exception to linting to ignore this folder

Reply:
I re-ran it and it added the unused generic imports. 
We could:
- Add some rules to remove those folders
- or
- Run the formatter after the scaffold command is complete

Reply:
I rather completely ignore an auto-generated folder than trying to "fix" it after each re-generation